### ISSUE TITLE
...medien.sch amdSec_10 false positive

### ISSUE THREAD

**Issue by jbarth-ubhd** [Deutsche-Digitale-Bibliothek/ddb-metadata-schematron-validation#10]
I assume `$work_dmdid` contains 1 "word" of the DMDID attribute ... example DMDID="w1 w2 w3" → $work_dmdid="w2",

but then... shouldn't `work_amdid` be found by something like `tokenize(@DMDID, " ") =any= $work_dmdid`  instead of  `@DMDID = $work_dmdid`  ?

```xml
<sch:let name="work_amdid"
value="//mets:mets/mets:structMap[@TYPE='LOGICAL']//mets:div[@DMDID = $work_dmdid]/@ADMID"/>
```

Example METS https://digi.ub.uni-heidelberg.de/cgi-bin/digioai.cgi?verb=GetRecord&metadataPrefix=mets&identifier=oai%3Adigi.ub.uni-heidelberg.de%3A26126

**Comment by tschleier** [comment]
Yes, that's correct - this is a bug.  
We'll fix it in the next version. Thank you for notifiying us.

**Comment by tschleier** [comment]
Fixed with 631fcdd6092fc1d0621501857ae3c1aac0afa96b
